src/python/pants/engine/internals/graph.py

```diff
@@ -261,7 +261,9 @@ async def addresses_with_origins_from_filesystem_specs(
             msg = (
                 f"No owning targets could be found for the file `{file_path}`.\n\nPlease check "
                 f"that there is a BUILD file in `{file_path.parent}` with a target whose `sources` field "
-                f"includes `{file_path}`. See https://www.pantsbuild.org/build_files.html."
+                f"includes `{file_path}`. See https://www.pantsbuild.org/build_files.html for more "
+                "information on target definitions.\n"
+                "If you would like to ignore un-owned files, please pass `--owners-not-found-behavior=ignore`."
```

Comment:
Should probably mention `warn` as a happy medium.

> If you would like to not error when files have no owners, please either use the global option `--owners-not-found-behavior=warn` or `--owners-not-found-behavior=ignore`.

Reply:
This message renders for warnings as well.